mlflow/utils/rest_utils.py

```diff
@@ -30,11 +32,24 @@
 )
 from mlflow.utils.string_utils import strip_suffix
 
+
 RESOURCE_NON_EXISTENT = "RESOURCE_DOES_NOT_EXIST"
 _REST_API_PATH_PREFIX = "/api/2.0"
 _TRACE_REST_API_PATH_PREFIX = f"{_REST_API_PATH_PREFIX}/mlflow/traces"
 
 
+@dataclass
+class _DatabricksSdkAPIErrorResponse:
+    status_code: int
+    reason: str
+    text: str
+    request = None
+    response = None
```

Comment:
what are the types of `request` and `response`?

Reply:
removed. we can't get `request` and `response` object here.